### PR TITLE
Making wasm location env-depended

### PR DESCRIPTION
**Pull request by Mexolius**

### CODE REVIEW

**Comment by Mexolius** [comment]
Hopefully this will fix both the undefined path issue and wasm access problems. If it doesn't fixes the wasm it will at least make it look for it in the same place as for the manifest.json and other assets from './public' dir

File: build_wasm.sh [hunk 1]
```diff
@@ -1,6 +1,5 @@
 PROJECT_NAME=weblibdedx
-PROPER_PATH="\`\${process.env.PUBLIC_URL}\/$PROJECT_NAME.wasm\`"
-BROKEN_PATH="wasmBinaryFile = \'weblibdedx.wasm\'"
+FIXED_PATH="\${process.env.PUBLIC_URL}\/$PROJECT_NAME.wasm"
```
Comment (grzanka) [inline build_wasm.sh:2]: What is ${process.env.PUBLIC_URL} ? I doubt that `build_wasm` would see the github action specific env variables.

I'd extract manipulation of this type completely from `build_wasm.sh` and move it to deploy stage (to github actions).

Imagine somebody wants to use this script in a cloned repository.

**Review by grzanka — CHANGES_REQUESTED** [review]
See comments inline

File: build_wasm.sh (hunk 1 above)
Comment (Mexolius) [reply build_wasm.sh:2]: Mind the "\" before the $. What it does is it changes the code to have that line in it. The value is read later when the code is run through react itself

Comment (Mexolius) [reply build_wasm.sh:2]: In other words the $ sign in this case is ment to be used as JavaScript string interpolation.

Comment (Mexolius) [reply build_wasm.sh:2]: Also take into account that the choice of the env variable isn't random - it's the same that's later responsible for loading the manifest.json and all assets in index.html (in the deployed builds)

Comment (grzanka) [reply build_wasm.sh:5]: OK, its clear now

**Review by grzanka — APPROVED** [review]
LGTM

**Pull request merged**